buildSrc/src/main/kotlin/JvmToolchain.kt

```diff
@@ -0,0 +1,14 @@
+import org.gradle.jvm.toolchain.JavaLanguageVersion
+import org.gradle.jvm.toolchain.JvmVendorSpec
+import org.gradle.api.Project
+
+private const val JDK_VERSION = 11
+
+val jvmToolchainVersion: JavaLanguageVersion
+    get() {
+        val jvmVersion = System.getProperty("PKL_JVM_VERSION", JDK_VERSION.toString()).toInt()
```

Comment:
[nit] unnecessary conversion from int to string back to int for the default case

```suggestion
        val jvmVersion = System.getProperty("PKL_JVM_VERSION")?.toInt() ?: JDK_VERSION
```